Formula/x/xz.rb

```diff
@@ -4,11 +4,11 @@ class Xz < Formula
   desc "General-purpose data compression with high compression ratio"
   homepage "https://tukaani.org/xz/"
   # The archive.org mirror below needs to be manually created at `archive.org`.
```

Comment:
@ik11235 We need to manually upload the tarball to archive.org. I've done that and pushed some changes here.

Reply:
👍

Reply:
@p-linnane 

Thanks for the advice!

I have confirmed the upload to archive.org and have reinstated the mirror.
I also confirmed that `brew test xz` succeeds in the local environment.